incognia.go

```diff
@@ -76,26 +77,32 @@ func New(config *IncogniaClientConfig) (*Client, error) {
 		return nil, ErrMissingClientIDOrClientSecret
 	}
 
-	if config.Timeout == 0 {
-		config.Timeout = time.Second * 10
+	var timeout time.Duration = config.Timeout
+	if timeout == 0 {
+		timeout = 10 * time.Second
```

Comment:
I think we should convert these timeouts (both here and at line 89) to constants.